<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="10" passed="5" failed="2" skipped="3">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-04-14T22:55:03 EDT" name="SolveProblem23" finished-at="2023-04-14T22:55:36 EDT" duration-ms="32742">
    <groups>
      <group name="solve-problem">
        <method signature="SolverTest.changeLanguageTest(java.lang.String)[pri:1, instance:com.devinarena.SolverTest@4efbca5a]" name="changeLanguageTest" class="com.devinarena.SolverTest"/>
        <method signature="SolverTest.clearPrevious()[pri:2, instance:com.devinarena.SolverTest@4efbca5a]" name="clearPrevious" class="com.devinarena.SolverTest"/>
        <method signature="SolverTest.failProblem()[pri:3, instance:com.devinarena.SolverTest@4efbca5a]" name="failProblem" class="com.devinarena.SolverTest"/>
        <method signature="SolverTest.attemptSolve(int)[pri:4, instance:com.devinarena.SolverTest@4efbca5a]" name="attemptSolve" class="com.devinarena.SolverTest"/>
      </group> <!-- solve-problem -->
      <group name="find-problem">
        <method signature="NavigateTest.navigateTest()[pri:1, instance:com.devinarena.NavigateTest@5c33f1a9]" name="navigateTest" class="com.devinarena.NavigateTest"/>
        <method signature="NavigateTest.loginTest()[pri:2, instance:com.devinarena.NavigateTest@5c33f1a9]" name="loginTest" class="com.devinarena.NavigateTest"/>
        <method signature="NavigateTest.findProblemTest(java.lang.String)[pri:3, instance:com.devinarena.NavigateTest@5c33f1a9]" name="findProblemTest" class="com.devinarena.NavigateTest"/>
      </group> <!-- find-problem -->
      <group name="login">
        <method signature="NavigateTest.navigateTest()[pri:1, instance:com.devinarena.NavigateTest@5c33f1a9]" name="navigateTest" class="com.devinarena.NavigateTest"/>
        <method signature="NavigateTest.loginTest()[pri:2, instance:com.devinarena.NavigateTest@5c33f1a9]" name="loginTest" class="com.devinarena.NavigateTest"/>
      </group> <!-- login -->
    </groups>
    <test started-at="2023-04-14T22:55:03 EDT" name="Problem23" finished-at="2023-04-14T22:55:36 EDT" duration-ms="32742">
      <class name="com.devinarena.NavigateTest">
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.devinarena.NavigateTest@5c33f1a9]" started-at="2023-04-14T22:55:03 EDT" name="setUp" finished-at="2023-04-14T22:55:05 EDT" duration-ms="1553" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="navigateTest()[pri:1, instance:com.devinarena.NavigateTest@5c33f1a9]" started-at="2023-04-14T22:55:05 EDT" name="navigateTest" finished-at="2023-04-14T22:55:09 EDT" duration-ms="4727" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- navigateTest -->
        <test-method signature="loginTest()[pri:2, instance:com.devinarena.NavigateTest@5c33f1a9]" started-at="2023-04-14T22:55:09 EDT" name="loginTest" finished-at="2023-04-14T22:55:16 EDT" duration-ms="6209" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTest -->
        <test-method signature="findProblemTest(java.lang.String)[pri:3, instance:com.devinarena.NavigateTest@5c33f1a9]" started-at="2023-04-14T22:55:16 EDT" name="findProblemTest" finished-at="2023-04-14T22:55:24 EDT" duration-ms="8693" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[23. Merge k Sorted Lists]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- findProblemTest -->
      </class> <!-- com.devinarena.NavigateTest -->
      <class name="com.devinarena.SignoutTest">
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.devinarena.SignoutTest@6069db50]" started-at="2023-04-14T22:55:36 EDT" name="setUp" finished-at="2023-04-14T22:55:36 EDT" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="signoutTest()[pri:1, instance:com.devinarena.SignoutTest@6069db50]" started-at="2023-04-14T22:55:36 EDT" name="signoutTest" depends-on-groups="solve-problem" finished-at="2023-04-14T22:55:36 EDT" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method SignoutTest.signoutTest()[pri:1, instance:com.devinarena.SignoutTest@6069db50] depends on not successfully finished methods in group "solve-problem"]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method SignoutTest.signoutTest()[pri:1, instance:com.devinarena.SignoutTest@6069db50] depends on not successfully finished methods in group "solve-problem"
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:150)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:284)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:119)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- signoutTest -->
        <test-method signature="confirmSignoutTest()[pri:3, instance:com.devinarena.SignoutTest@6069db50]" started-at="2023-04-14T22:55:36 EDT" name="confirmSignoutTest" depends-on-groups="solve-problem" finished-at="2023-04-14T22:55:36 EDT" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method SignoutTest.confirmSignoutTest()[pri:3, instance:com.devinarena.SignoutTest@6069db50] depends on not successfully finished methods in group "solve-problem"]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method SignoutTest.confirmSignoutTest()[pri:3, instance:com.devinarena.SignoutTest@6069db50] depends on not successfully finished methods in group "solve-problem"
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:150)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:284)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:119)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- confirmSignoutTest -->
        <test-method signature="screenshotPageTest()[pri:2, instance:com.devinarena.SignoutTest@6069db50]" started-at="2023-04-14T22:55:36 EDT" name="screenshotPageTest" depends-on-groups="solve-problem" finished-at="2023-04-14T22:55:36 EDT" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method SignoutTest.screenshotPageTest()[pri:2, instance:com.devinarena.SignoutTest@6069db50] depends on not successfully finished methods in group "solve-problem"]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method SignoutTest.screenshotPageTest()[pri:2, instance:com.devinarena.SignoutTest@6069db50] depends on not successfully finished methods in group "solve-problem"
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:150)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:284)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:119)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- screenshotPageTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.devinarena.SignoutTest@6069db50]" started-at="2023-04-14T22:55:36 EDT" name="tearDown" finished-at="2023-04-14T22:55:36 EDT" duration-ms="121" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.devinarena.SignoutTest -->
      <class name="com.devinarena.SolverTest">
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.devinarena.SolverTest@4efbca5a]" started-at="2023-04-14T22:55:24 EDT" name="setUp" finished-at="2023-04-14T22:55:24 EDT" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="changeLanguageTest(java.lang.String)[pri:1, instance:com.devinarena.SolverTest@4efbca5a]" started-at="2023-04-14T22:55:24 EDT" name="changeLanguageTest" depends-on-groups="find-problem" finished-at="2023-04-14T22:55:26 EDT" duration-ms="1636" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[python]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- changeLanguageTest -->
        <test-method signature="clearPrevious()[pri:2, instance:com.devinarena.SolverTest@4efbca5a]" started-at="2023-04-14T22:55:26 EDT" name="clearPrevious" depends-on-groups="find-problem" finished-at="2023-04-14T22:55:28 EDT" duration-ms="2129" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- clearPrevious -->
        <test-method signature="failProblem()[pri:3, instance:com.devinarena.SolverTest@4efbca5a]" started-at="2023-04-14T22:55:28 EDT" name="failProblem" depends-on-groups="find-problem" finished-at="2023-04-14T22:55:33 EDT" duration-ms="5068" status="FAIL">
          <exception class="org.openqa.selenium.NoSuchElementException">
            <message>
              <![CDATA[no such element: Unable to locate element: {"method":"xpath","selector":"//*[@id="qd-content"]/div[3]/div/div[3]/div/div/div[1]/div/div[3]/div[4]"}
  (Session info: chrome=111.0.5563.147)
For documentation on this error, please visit: https://selenium.dev/exceptions/#no_such_element
Build info: version: '4.8.3', revision: 'e5e76298c3'
System info: os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.2'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [c86b4d815f7a5f33917616f0a880e5fd, findElement {using=xpath, value=//*[@id="qd-content"]/div[3]/div/div[3]/div/div/div[1]/div/div[3]/div[4]}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 111.0.5563.147, chrome: {chromedriverVersion: 111.0.5563.64 (c710e93d5b63..., userDataDir: C:\Users\devin\AppData\Loca...}, goog:chromeOptions: {debuggerAddress: localhost:49597}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: WINDOWS, proxy: Proxy(), se:cdp: ws://localhost:49597/devtoo..., se:cdpVersion: 111.0.5563.147, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: c86b4d815f7a5f33917616f0a880e5fd]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.NoSuchElementException: no such element: Unable to locate element: {"method":"xpath","selector":"//*[@id="qd-content"]/div[3]/div/div[3]/div/div/div[1]/div/div[3]/div[4]"}
  (Session info: chrome=111.0.5563.147)
For documentation on this error, please visit: https://selenium.dev/exceptions/#no_such_element
Build info: version: '4.8.3', revision: 'e5e76298c3'
System info: os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.2'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [c86b4d815f7a5f33917616f0a880e5fd, findElement {using=xpath, value=//*[@id="qd-content"]/div[3]/div/div[3]/div/div/div[1]/div/div[3]/div[4]}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 111.0.5563.147, chrome: {chromedriverVersion: 111.0.5563.64 (c710e93d5b63..., userDataDir: C:\Users\devin\AppData\Loca...}, goog:chromeOptions: {debuggerAddress: localhost:49597}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: WINDOWS, proxy: Proxy(), se:cdp: ws://localhost:49597/devtoo..., se:cdpVersion: 111.0.5563.147, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: c86b4d815f7a5f33917616f0a880e5fd
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:200)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:133)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:53)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:193)
at org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:183)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:158)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:543)
at org.openqa.selenium.remote.ElementLocation$ElementFinder$2.findElement(ElementLocation.java:162)
at org.openqa.selenium.remote.ElementLocation.findElement(ElementLocation.java:60)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:352)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:344)
at com.devinarena.LeetCoder.getElement(LeetCoder.java:40)
at com.devinarena.LeetCoder.click(LeetCoder.java:58)
at com.devinarena.SolverTest.failProblem(SolverTest.java:56)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:284)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:119)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.NoSuchElementException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- failProblem -->
        <test-method signature="attemptSolve(int)[pri:4, instance:com.devinarena.SolverTest@4efbca5a]" started-at="2023-04-14T22:55:33 EDT" name="attemptSolve" depends-on-groups="find-problem" finished-at="2023-04-14T22:55:36 EDT" duration-ms="2562" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[23]]>
              </value>
            </param>
          </params>
          <exception class="org.openqa.selenium.NoSuchWindowException">
            <message>
              <![CDATA[no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=111.0.5563.147)
Build info: version: '4.8.3', revision: 'e5e76298c3'
System info: os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.2'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [c86b4d815f7a5f33917616f0a880e5fd, findElement {using=xpath, value=//*[@id="qd-content"]/div[3]}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 111.0.5563.147, chrome: {chromedriverVersion: 111.0.5563.64 (c710e93d5b63..., userDataDir: C:\Users\devin\AppData\Loca...}, goog:chromeOptions: {debuggerAddress: localhost:49597}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: WINDOWS, proxy: Proxy(), se:cdp: ws://localhost:49597/devtoo..., se:cdpVersion: 111.0.5563.147, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: c86b4d815f7a5f33917616f0a880e5fd]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.NoSuchWindowException: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=111.0.5563.147)
Build info: version: '4.8.3', revision: 'e5e76298c3'
System info: os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.2'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [c86b4d815f7a5f33917616f0a880e5fd, findElement {using=xpath, value=//*[@id="qd-content"]/div[3]}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 111.0.5563.147, chrome: {chromedriverVersion: 111.0.5563.64 (c710e93d5b63..., userDataDir: C:\Users\devin\AppData\Loca...}, goog:chromeOptions: {debuggerAddress: localhost:49597}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: WINDOWS, proxy: Proxy(), se:cdp: ws://localhost:49597/devtoo..., se:cdpVersion: 111.0.5563.147, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: c86b4d815f7a5f33917616f0a880e5fd
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:200)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:133)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:53)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:193)
at org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:183)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:158)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:543)
at org.openqa.selenium.remote.ElementLocation$ElementFinder$2.findElement(ElementLocation.java:162)
at org.openqa.selenium.remote.ElementLocation.findElement(ElementLocation.java:60)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:352)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:344)
at com.devinarena.LeetCoder.getElement(LeetCoder.java:40)
at com.devinarena.LeetCoder.click(LeetCoder.java:58)
at com.devinarena.SolverTest.attemptSolve(SolverTest.java:72)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:284)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:119)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.NoSuchWindowException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- attemptSolve -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.devinarena.SolverTest@4efbca5a]" started-at="2023-04-14T22:55:36 EDT" name="tearDown" finished-at="2023-04-14T22:55:36 EDT" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.devinarena.SolverTest -->
    </test> <!-- Problem23 -->
  </suite> <!-- SolveProblem23 -->
</testng-results>
